### DISCUSSION TITLE
Multiple configuration files

### DISCUSSION THREAD

**Discussion by phhoef** [quarkusio/quarkus#24385] [Q&A] [answered]
Hi, I am very new to quarkus and started creating my app. 
The app is highly configurable and I plan to use yml as format. 
to keep the configurations clear, I would like to split it up in multiple configuration files. 

From what I saw in the tutorials, it’s not a common use case. 
is there a possibility to have several (even dynamic) configuration files with the same validation and type safety like usual?

**Comment by gsmet** [comment, marked as answer]
Hi!

You should be able to achieve what you want by defining additional configuration files in `quarkus.config.locations`.
